src/views/Items/Item.js

```diff
@@ -0,0 +1,43 @@
+import React, {Component, PropTypes} from 'react';
+import classNames from 'classnames';
+
+export default class Item extends Component {
+  static propTypes = {
+    children: PropTypes.node,
+    className: PropTypes.string,
+    description: PropTypes.node,
+    extra: PropTypes.node,
+    heading: PropTypes.node,
+    image: PropTypes.node,
+    meta: PropTypes.node,
+  };
```

Comment:
Pass in all "pieces" of the component as props to make sure that all content matches semantic's markup.